src/hip_texture.cpp

```diff
@@ -381,7 +381,7 @@ hipError_t hipGetTextureObjectTextureDesc(hipTextureDesc* pTexDesc,
 }
 
 // Texture Reference APIs
-hipError_t ihipBindTextureImpl(int dim, enum hipTextureReadMode readMode, size_t* offset,
+hipError_t ihipBindTextureImpl(TlsData *tls, int dim, enum hipTextureReadMode readMode, size_t* offset,
```

Comment:
```ihipBindTextureImpl``` is used by the cplusplus variants of ```hipBindTexture``` defined in include/hip/hcc_detail/hip_runtime_api.h. Hence this change breaks the unit test directed_test/texture/hipBindTexRef1DFetch.